package.json

```diff
@@ -8,6 +8,7 @@
     "dev-server": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --port 2992 --inline",
     "hot-dev-server": "webpack-dev-server --config webpack-hot-dev-server.config.js --hot --progress --colors --port 2992 --inline",
     "build": "webpack --config webpack-production.config.js --progress --profile --colors",
+    "build-static": "webpack --config webpack-static-site.config.js --progress --profile --colors && node gh-pages-deploy.js",
```

Comment:
could you split it into two tasks. `build-static` and `deploy-static`. Maybe someone want to test the static page before deploying... `deplay-static` should call `npm run build-static`